src/NetAppFiles/NetAppFiles/help/Update-AzNetAppFilesPool.md

```diff
@@ -0,0 +1,243 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.NetAppFiles.dll-Help.xml
+Module Name: Az.NetAppFiles
+online version: https://docs.microsoft.com/en-us/powershell/module/az.netappfiles/update-aznetappfilespool
+schema: 2.0.0
+---
+
+# Update-AzNetAppFilesPool
+
+## SYNOPSIS
+Updates an Azure NetApp Files (ANF) pool with the new data set.
+
+## SYNTAX
+
+### ByFieldsParameterSet (Default)
+```
+Update-AzNetAppFilesPool -ResourceGroupName <String> -Location <String> -AccountName <String> -Name <String>
+ -PoolSize <Int64> -ServiceLevel <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
```

Comment:
Why are these mandatory? Shouldn't they be optional?

Reply:
This is a special case which was explained early on in the review process. The patch for pool is actually missing and put is used. This is corrected in the newer SDK which will feed to an update of the PowerShell in a few weeks.